extensions/gitpod-web/webview-ui/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "webview-ui",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"build": "rollup -c",
+		"dev": "rollup -c -w",
+		"start": "sirv public --no-clear",
+		"check": "svelte-check --tsconfig ./tsconfig.json"
+	},
+	"dependencies": {
+		"@vscode/codicons": "^0.0.31",
+		"@vscode/webview-ui-toolkit": "^1.0.0",
+		"sirv-cli": "^2.0.0"
+	},
+	"devDependencies": {
+		"@rollup/plugin-commonjs": "^17.0.0",
+		"@rollup/plugin-json": "^4.1.0",
+		"@rollup/plugin-node-resolve": "^11.0.0",
+		"@rollup/plugin-typescript": "^8.0.0",
+		"@tsconfig/svelte": "^2.0.0",
+		"@types/vscode-webview": "^1.57.0",
+		"rollup": "^2.3.4",
+		"rollup-plugin-copy": "^3.4.0",
+		"rollup-plugin-css-only": "^3.1.0",
+		"rollup-plugin-livereload": "^2.0.0",
+		"rollup-plugin-svelte": "^7.0.0",
+		"rollup-plugin-terser": "^7.0.0",
+		"svelte": "^3.0.0",
+		"svelte-check": "^2.0.0",
+		"svelte-preprocess": "^4.0.0",
+		"tslib": "^2.0.0",
+		"typescript": "^4.0.0"
+	}
+}
```

Comment:
Could we just use one package.json? I see other extensions that also have webviews and only have one `package.json`

Reply:
No, this file structure is learn from an official demo. It makes things (both structure and code review) easier than before, I'll keep it.

Reply:
File structure is fine, I meant to just move `package.json` content i.e `scripts`, `dependencies`, `devDependencies` to the parent package.json so there's only one `yarn.lock` file for gitpod-web extension

Reply:
We need dependencies like `@vscode/codicons` `@vscode/webview-ui-toolkit` to install in `current folder node_moddules`. with `extensions/` folder's setup, some of them will be inside `extensions/node_modules` this is very bad for the build, and I don't want this to happen again (configure of `nohoist` sometimes make things complex).

And these PKGs will truly work on `webview-ui` only, after `webview-ui` build success, we don't need them anymore.

Everything related to webview is only existed in `webview-ui` folder, I think that's good, it only works to build what extension needs

Reply:
I don't know if it's worth moving `scripts/dep/dev-dep` out, since other extensions with webview don't use front-framework

Reply:
What is `scripts/dep/dev-dep`?

Reply:
scripts, dependencies, devDependencies

Reply:
I added `nohoist` back excluding the gitpod, grpc packages that caused the previous issue, could you rebase and test it :pray: